platform/linux-generic/odp_queue.c

```diff
@@ -89,10 +89,10 @@ static int queue_init(queue_entry_t *queue, const char *name,
 	}
 	queue->s.type = queue->s.param.type;
 
-	queue->s.enqueue = queue_enq;
-	queue->s.dequeue = queue_deq;
-	queue->s.enqueue_multi = queue_enq_multi;
-	queue->s.dequeue_multi = queue_deq_multi;
+	queue->s.enqueue = _queue_enq;
+	queue->s.dequeue = _queue_deq;
+	queue->s.enqueue_multi = _queue_enq_multi;
+	queue->s.dequeue_multi = _queue_deq_multi;
```

Comment:
I do not plan to rename for this patch. This needs to be inline with scheduler (scheduler does not have **internal_** prefixes). I plan on making a separate doc/patch to more streamline these things for odp-cloud(we need to address optimized static selection as well).